operator/config/overlays/community-openshift/manager_security_context_patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          securityContext:
+            capabilities:
+              drop:
+                - ALL
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+           type: RuntimeDefault
```

Comment:
You don't need to add this patch in the `community-openshift` overlay if exactly the same exists in the `openshift` overlay. The first is pulling the latter, see kustomization.yaml.